File: Amigo.Tenant.Application.DTOs/Responses/Houses/HouseServicePeriodDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Amigo.Tenant.Application.DTOs.Responses.Houses
{
    public class HouseServicePeriodDTO
    {
        public int HouseServicePeriodId { get; set; }
        public int HouseServiceId { get; set; }
        public int MonthId { get; set; }
        public int DueDateMonth { get; set; }
        public int DueDateDay { get; set; }
        public int CutOffMonth { get; set; }
        public int CutOffDay { get; set; }

        public bool RowStatus { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreationDate { get; set; }
        public int UpdatedBy { get; set; }
        public DateTime UpdatedDate { get; set; }

        public int? PeriodId { get; set; }
        public decimal? Amount { get; set; }
        public decimal? Adjust { get; set; }

        public decimal? Consumption { get; set; }
        public int? ConsumptionUnmId { get; set; }
        public int? HouseServicePeriodStatusId { get; set; }
    }
}

File: Amigo.Tenant.UI.Web/src/app/shared/dtos/House/HouseServicePeriodDTO.ts
export class HouseServicePeriodDTO {
    public houseServicePeriodId: number;
    public houseServiceId: number;

    public monthId: number | null;
    public dueDateMonth: number;
    public dueDateDay: number;
    public cutOffMonth: number;
    public cutOffDay: number;
    
    public month: string;

    public rowStatus: boolean | null;
    public createdBy: number | null;
    public creationDate: Date | null;
    public updatedBy: number | null;
    public updatedDate: Date | null;
}
